### ISSUE TITLE
plz watch fails with "Failed to acquire lock: bad file descriptor"

### ISSUE THREAD

**Issue by mvgijssel** [thought-machine/please#1981]
When I run `plz watch` with either a test or a run target the command immediately fails with:

```
12:28:43.421 WARNING: Looks like another plz is already running in this repo. Waiting for it to finish...
12:28:43.421 CRITICAL: Failed to acquire lock: bad file descriptor
```

Running `plz watch` on a build target looks like it works by outputting something like this:

```
Build finished; total time 80ms, incrementality 100.0%. Outputs:
//tools/template:template:
  plz-out/gen/tools/template/template.build_defs
And now my watch begins...
```

But the moment I save a file that's being watched it crashes with the same two log lines. 

Building, testing and running works fine without the watch command.

My environment:

```
OS: macos 11.4
plz --version: Please version 16.2.1
```

**Comment by Tatskaari** [comment]
This was fixed in 16.3.0 :) 

https://github.com/thought-machine/please/pull/1944

**Comment by mvgijssel** [comment]
Thanks! ❤️
